__tests__/app.test.js

```diff
@@ -55,3 +55,32 @@ describe('GET /api', () => {
 
     })
 })
+
+describe('GET /api/articles/:article_id', () => {
+    test('status:200, responds with a JSON object containing specified article which has the following properties: author, title, article_id, body, topic, created_at, votes, article_img_url', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {
+
+        expect(body.article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String)
+            })
+        })
+    })
+    test('status:404, responds with "Not Found" when article_id does not exist', () => {
+        return request(app)
+        .delete('/api/articles/9999')
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe('Not Found')
+        })
+    })
```

Comment:
This is a great start to the error handling tests as this test is ensuring if a valid id is given but simply does not exist the user should be given an appropriate status and message, but what if the user gave a completely invalid id? what should the user be given then?